server/src/api/modules/groups/group.utils.ts

```diff
@@ -16,4 +19,19 @@ function buildDefaultSocialLinks() {
   };
 }
 
-export { sanitizeName, buildDefaultSocialLinks };
+function sortMembers(group: Omit<GroupDetails, 'socialLinks' | 'memberCount'>): MembershipWithPlayer[] {
```

Comment:
One last nitpick:

Could you change this to `sortMembers(memberships: MembershipWithPlayer[], roleOrders?: GroupRoleOrder[])` instead of receiving the whole group details object and doing an omit type on it?

Reply:
done